### PR TITLE
scripts&commands; small fix of george, cnc, dispel's implementations;

### PR DESCRIPTION
**Pull request by YihaoXue**

### CODE REVIEW

File: src/spuco/robust_train/cnc_train.py [hunk 1]
```diff
@@ -180,6 +180,7 @@ def __init__(
         self.sampler = sampler
         self.verbose = verbose
         self.device = device
+        self.name = 'CnC'
```
Comment (sjoshi804) [inline src/spuco/robust_train/cnc_train.py:183]: what is this for?

File: src/spuco/last_layer_retrain/dispel.py [hunk 2]
```diff
@@ -20,6 +20,7 @@ def __init__(
         labeled_valset_size: float = 0.5,
         C_range: List[float] = [1., 0.7, 0.3, 0.1, 0.07, 0.03, 0.01],
         s_range: List[float] = [1.0, 0.9, 0.8, 0.7],
+        groups_with_spu: Optional[List[int]] = None,
```
Comment (sjoshi804) [inline src/spuco/last_layer_retrain/dispel.py:23]: what is this?

Comment (sjoshi804) [reply src/spuco/last_layer_retrain/dispel.py:23]: needs to be made clearer since we we want to expose this API to users.

Comment (YihaoXue) [reply src/spuco/last_layer_retrain/dispel.py:23]: the current base class requires self.name

File: src/spuco/robust_train/cnc_train.py (hunk 1 above)
Comment (YihaoXue) [reply src/spuco/robust_train/cnc_train.py:183]: the current base class requires self.name

File: src/spuco/last_layer_retrain/dispel.py (hunk 2 above)
Comment (YihaoXue) [reply src/spuco/last_layer_retrain/dispel.py:23]: this is for cases where there is a missing group. then pass the existing groups that share the spu feature with the missing group to this variable. this is for the study in my paper

**Pull request merged**